hw/ip/otbn/data/insns.yml

```diff
@@ -910,6 +921,31 @@ insns:
         bodysize: bodysize
         iterations: iterations
 
+  - mnemonic: nop
+    synopsis: No operation
+    rv32i: true
+    operands: []
+    doc: A pseudo-operation that has no effect.
+    literal-pseudo-op:
+      - addi x0, x0, 0
+
+  - mnemonic: li
+    synopsis: Load immediate
+    rv32i: true
+    operands: [grd, imm]
+    doc: |
+      Load a 32-bit immediate value into a GPR. Expands to one or two
+      instructions, depending on the immediate.
+    python-pseudo-op: true
+
+  - mnemonic: ret
+    synopsis: Return from subroutine
+    rv32i: true
+    operands: []
+    doc: A pseudo-operation that has no effect.
+    literal-pseudo-op:
+      - jalr x0, x1, 0
```

Comment:
Hmm, yes. I guess I have a personal preference to avoid SHOUTING (we're not writing fortran 77 any more!), but I agree that this needs to be capitalised for consistency :-(